src/test/java/teammates/test/cases/ui/browsertests/AdminHomePageUiTest.java

```diff
@@ -223,8 +223,9 @@ private void testCreateInstructorAction() throws Exception {
 
         ______TS("new instructor can click submit button of sample feedbackSession");
         instructorHomePage.loadInstructorHomeTab();
-        FeedbackSubmitPage fbsp = instructorHomePage.clickFeedbackSessionSubmitLink("AHPUiT.instr1.gma-demo",
-                                                                                    "Second team feedback session");
+        FeedbackSubmitPage fbsp =
+                instructorHomePage.clickFeedbackSessionSubmitLink("AHPUiT.instr1.gma-demo",
+                                                                  "Second team feedback session");
```

Comment:
Same as above:
Maybe put the method call on the first line, and put the arguments on the second line.